### ISSUE TITLE
Build 165 windows installer, can't process load in browser menu after clicking Start Server menu.

### ISSUE THREAD

**Issue by djallado** [learningequality/installers#84]
I tested windows installer **build 165** in `IE8 - Windows7.`

**My steps:**
1. Install IE8 - Windows7 in virtual box
2. Download windows installer **build 165** from `dungeon`
3. Install `KALiteSetup-0.14.dev14.exe` *(latest installer downloaded from dungeon)*
4. Follow `KA Lite Setup Wizard`
5. Launch `KA Lite`
6. Right click `KA Lite` logo in the `System Tray`.
![start](https://cloud.githubusercontent.com/assets/8664071/8507884/4921e252-2289-11e5-9fd2-c3796913345b.png)
7. Click `Start server` and wait for a few minutes.
8. Right click `KA Lite` logo in the `System Tray`, The option `Load in a browser` and `Stop server` are greyed. 
![greyed](https://cloud.githubusercontent.com/assets/8664071/8507886/535a7a22-2289-11e5-8392-d31d306f5ce3.png)

 
The latest installer is not working in `IE8 - Windows7`. 
The same steps I did in `IE11 - Windows7` setup and still not working.
I also observed the `.kalite` directory, I attached the screenshot:
![dotkalite](https://cloud.githubusercontent.com/assets/8664071/8507875/0ff68b86-2289-11e5-9df3-deda2052ab22.png)

Notice that there's no server log in the `.kalite` directory.

**Comment by djallado** [comment]
@aronasorman and @EdDixon.

I'm testing `Windows Installer build 183` in `IE10 - Windows7` fresh install.
The installer is not working.
After I launched `KA Lite` and then clicked `Start Server` in `System Tray` , the `Load into browser` menu still greyed even I waited for more than 10 minutes.

I checked pip the dependencies, there's no `ka-lite-static-0.14.0` like in the screenshot:
![latest](https://cloud.githubusercontent.com/assets/8664071/8647735/36c71216-298d-11e5-9b5e-4163e908fe13.png)

To make it work, I have to install `kalite-static` manually like screenshot below:
![after_install_static](https://cloud.githubusercontent.com/assets/8664071/8647905/b4932580-298e-11e5-80f0-fdac0acf00e6.png)

Then when I launch KA-Lite, it works fine. I can now download videos like in a screenshot below:
![123](https://cloud.githubusercontent.com/assets/8664071/8648239/976ce6fa-2991-11e5-9ba8-f43295cb0dd5.png)

In some cases like @amodia, He installed `KA Lite` installer from this link: http://field.learningequality.org/KALiteSetup-0.14.0.exe first. Then when he tried uninstall `KA Lite installer` and install again some installer from `dungeon` it works properly. That's because the `ka-lite-static` is already been installed, so it works in the new installer. But when we try it install in a fresh `Window OS`, the latest build is not working.
